server/routes/overview/overviewView.ts

```diff
@@ -111,32 +66,6 @@ export default class OverviewView {
       prisonerHasHadInduction && !this.actionPlanReviews?.problemRetrievingData && mostRecentReviewSession == null
     const prisonerHasHadInductionAndAtLeastOneReview = prisonerHasHadInduction && mostRecentReviewSession != null
 
-    const reviewScheduleDoesNotExist = this.actionPlanReviews?.latestReviewSchedule == null
```

Comment:
This logic, and the similar new logic for working out the Induction Schedule stuff, has been moved to `overviewViewFunctions.ts`